src/components/AutoSuggestion/AutoSuggestion.tsx

```diff
@@ -471,6 +496,14 @@ export const AutoSuggestion = ({
                 default:
                 //do nothing
             }
+        } else {
+            if (multiline && editorState.editorInstance && keyPressedFromInput === OVERWRITTEN_KEYS.Enter) {
```

Comment:
This does not seem to work reliably, e.g. by enter random chars quickly followed by pressing the Enter key does not always create a newline. 
Shouldn't this be handled in `handleInputEditorKeyPress` instead by not swallowing the ENTER key press?
There seems to be a UX conflict here. The arrow and Enter keys have other semantics in the multiline case compared to the single line editor case.